src/Components/benchmarkapps/Wasm.Performance/TestApp/Pages/Json.razor

```diff
@@ -55,12 +58,18 @@
     void SerializeLarge()
         => serializedValue = JsonSerializer.Serialize(largeOrgChart);
 
+    void SerializeLargeSourceGen()
+        => serializedValue = JsonSerializer.Serialize(largeOrgChart, personJsonContext.Person);
```

Comment:
Why aren't we using `SerializeToUtf8Bytes` (and `byte[]` on deserialize) since that's the serializer's native encoding?

Reply:
I think these benchmarks are meant to represent the sort of code we expect customers to write. `JsonSerializer.Serialize` would be a more commonly-used API, and internally makes use of most of the same code paths as `SerializeToUtf8Bytes`, right?

Reply:
That's true - they use most of the same code-paths but `string`-based overloads also incur the cost of transcoding. We can consider this a constant cost of using these APIs, so no need to change here.